lib/src/radio/Radio.stories.tsx

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+import DxcRadio from "./Radio";
+import { BackgroundColorProvider } from "../BackgroundColorContext";
+import Title from "../../.storybook/components/Title";
+import ExampleContainer from "../../.storybook/components/ExampleContainer";
+import DarkContainer from "../../.storybook/components/DarkSection";
+import { userEvent, within } from "@storybook/testing-library";
+
+export default {
+  title: "Radio",
+  component: DxcRadio,
+};
+
+const Radio = () => (
+  <>
+    <>
+      <ExampleContainer>
+        <Title title="Default" theme="light" level={4} />
+        <DxcRadio label="Radio" />
+      </ExampleContainer>
+      <ExampleContainer pseudoState="pseudo-focus">
```

Comment:
If we are simulating getting the focus through tabulation with userEvent, I think this pseudoState could be deleted.

Reply:
Done! ✌🏼